assets/js/metric_config.js

```diff
@@ -0,0 +1,380 @@
+  import * as echarts from  './echarts.min.js';
```

Comment:
I think you should use npm package if there is

Reply:
hi ,
I corrected my error in latest commit